cpp/benchmarks/common/generate_input.hpp

```diff
@@ -359,6 +372,23 @@ class data_profile {
     }
   }
 
+  // Users should pass integral values for bounds when setting the parameters for fixed-point.
+  // Otherwise the call with have no effect.
+  template <typename T,
+            typename Type_enum,
+            std::enable_if_t<cuda::std::is_integral_v<T>, T>* = nullptr>
```

Comment:
Is `CUDF_ENABLE_IF` available in `benchmarks/`?  If yes, then that might be slightly more readable:
```suggestion
            CUDF_ENABLE_IF(cuda::std::is_integral_v<T>)
```

Reply:
Oh man, that looks super useful.  Unfortunately traits.hpp includes fixed_point.hpp because it introduces is_fixed_point().

Reply:
Thank you for clarifying, @pmattione-nvidia.  That's a shame.